ocean_provider/utils/test/test_util.py

```diff
@@ -139,7 +139,7 @@ class Dummy:
 
 def test_download_ipfs_file(client):
     cid = "QmQfpdcMWnLTXKKW9GPV7NgtEugghgD6HgzSF6gSrp2mL9"
-    url = f"ipfs://{cid}"
+    url = f"http://ipfs.io/ipfs/{cid}"
```

Comment:
this is not right. This is a test for a http-resolved ipfs URL. We want to test that an ipfs:// file is resolved and downloaded right.